core/dbt/tests/util.py

```diff
@@ -73,7 +73,8 @@ def run_dbt(args: List[str] = None, expect_pass=True):
         args = ["run"]
 
     print("\n\nInvoking dbt with {}".format(args))
-    res, success = handle_and_check(args)
+    dbt = dbtRunner()
```

Comment:
I love how simple this change ended up being.  It's a sign that we made the right abstractions when we implemented the new functional testing framework.